### PR TITLE
Add voro++

### PR DESCRIPTION
**Pull request by ltalirz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ltalirz** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by ltalirz** [comment]
the osx build fails but it seems to me this is a conda-forge issue (?) - at least I've seen the same error in a couple of other recent open PRs as well.

I plan on adding a windows build in the feedstock later; I've disabled it for the time being since I couldn't figure out the windows analogue of creating the static library [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=307884&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=8562) (no experience on windows, and no access to a windows machine)
```
ar rs libvoro++.a cell.o common.o container.o unitcell.o v_compute.o c_loops.o v_base.o wall.o pre_container.o container_prd.o
```

**Comment by ltalirz** [comment]
@conda-forge-admin, please restart ci

**Comment by ltalirz** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by ltalirz** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/voro/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+mkdir -p ${PREFIX}/lib
+
+make
+make install
```
Comment (ocefpaf) [inline recipes/voro/build.sh:7]: Probably not but I have to ask. Is there a `make check` that we can run here?

Comment (ltalirz) [reply recipes/voro/build.sh:7]: thanks for the review - no , [there isn't yet](https://github.com/chr1shr/voro/blob/master/Makefile#L8) but I did ask for creating something like a test suite here
https://github.com/chr1shr/voro/issues/8#issuecomment-826649043

It should be straightforward but the developer currently seems busy

**Pull request merged**